### PR TITLE
Fix accesibility nav-tabs issues

### PR DESCRIPTION
**Pull request by Vowoo**
Due to issue #2551 I fix the accesibility nav-tabs anchor

Here is a video about the issue fixed: 

https://user-images.githubusercontent.com/57190230/162172928-0877c557-d077-45d9-b5d8-378d44533d6d.mp4

### CODE REVIEW

**Comment by MikeMcC399** [comment]
I was a bit confused to see http://localhost:8001/en/accessibility/#glossary in the address bar when clicking either Android or iOS. 

This is a different experience compared to https://www.coronawarn.app/en/screenshots/ which shows  https://www.coronawarn.app/en/screenshots/#android_screenshots or https://www.coronawarn.app/en/screenshots/#ios_screenshots when switching between the operating systems.

File: src/assets/js/app.js [hunk 1]
```diff
@@ -904,20 +904,29 @@ $(document).ready(function(){
...
         }
       });
 
+    $('.mobile-type').ready(function(e){
+        if(location.hash === "#ios" || location.hash === "#android"){
+            $(window).scrollTop($("#glossary").offset().top);
+        }
+    });
+    $(window).bind( 'hashchange', function(e) { 
+        console.log("he sido cambiado")
```
Comment (MikeMcC399) [inline src/assets/js/app.js:920]: If this is intended as a permanent change then the text should be in ingles!

Comment (Vowoo) [reply src/assets/js/app.js]: yes sorry I was making some testing, now i have pushed a new commit.
thank you, see you tomorrow!

**Review by MikeMcC399 — APPROVED** [review]
This looks fine to me now.

**Review by larswmh — APPROVED** [review]

**Review by dsarkar — APPROVED** [review]

**Comment by dsarkar** [comment]
Morning @MikeMcC399 Thanks for reviewing and testing!

**Pull request merged**